prover/witness_generator/src/main.rs

```diff
@@ -39,7 +39,10 @@ struct Opt {
     batch_size: Option<usize>,
     /// aggregation round for the witness generator.
     #[structopt(short = "r", long = "round")]
-    round: AggregationRound,
+    round: Option<AggregationRound>,
+    /// start all aggregation rounds for the witness generator.
```

Comment:
please add more descriptive comments -- mention that the binary can either be run for a single aggregation round, or for all rounds.

Reply:
I added this line to the Opt struct: `Aggregation rounds options, they can be run individually or together.` https://github.com/matter-labs/zksync-era/pull/258/commits/3435d1b8ce22056cad473965b937e3e7f59a1c72